src/locales/en-US/admin.ftl

```diff
@@ -1243,14 +1245,19 @@ community-filter-statusSelectField =
 community-changeRoleButton =
   .aria-label = Change role
 
-community-assignMySites = Assign my sites
-community-removeMySites = Remove my sites
+community-assignMySitesToModerator = Assign my sites to moderator
+community-removeMySitesFromModerator = Remove my sites from moderator
+community-assignMySitesToMember = Assign my sites to member
+community-removeMySitesFromMember = Remove my sites from member
 community-stillHaveSiteModeratorPrivileges = They will still have Site Moderator privileges for:
+community-stillHAveMemberPriviledges = They will still have Member priviledges for:
```

Comment:
This translation key doesn't match the one in the file. It looks like it should be `community-stillHaveMemberPrivileges` instead.